src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionForm.tag

```diff
@@ -47,7 +47,10 @@
                     There are no questions for you to answer here!
             </c:when>
             <c:otherwise>
-                <input type="submit" class="btn btn-primary center-block"
+                <input type="checkbox" name="isSendSubmissionEmail" value="true" checked>
+                    Send me a confirmation email
+                </input>
```

Comment:
The `input` element is misconfigured here:
- It is a void element, i.e there should be no closing `</input>`
- The value "true" is unnecessary

And also rename `isSendSubmissionEmail` as per my other comment.